TUTORIAL_USE.md

```diff
@@ -0,0 +1,118 @@
+# Using the existing code generators
+
+Java::Geci provides several out-of-the-box code generators. 
+In this tutorial we'll use the generator `Accessor` as an example, 
+but the things explained in this tutorial apply to every other generator.
+
+You use Java::Geci by using the `Geci` class in tests.
+
+```java
+public class TestAccessor {
+
+    @Test
+    public void testAccessor() throws Exception {
+        Assertions.assertFalse(new Geci().source(maven()
+                        .module("javageci-examples").mainSource())
+                        .register(Accessor.builder().build()).generate(),
+                Geci.FAILED);
+    }
+}
+```
+This might seem like a lot at first, so let's break it down.
+
+`Geci` has a fluent interface.
+
+The method `source()` can be used to specify the directories where your
+source files are. If you have source files in different places you
+have to chain several `source()` invocations one after the other. Every
+single call to `source()` can specify several directories. These count
+as one single directory and the first that exists is used to discover
+the files.<br/>
+This means:
+* Call `source("foo").source("bar")` if you have sources in **both** foo **and** bar.
+* Call `source("foo", "bar")` if you have sources in **either** foo **or** bar.
+*maven().module("javageci-examples").mainSource()* is just "javageci-examples/src"
```

Comment:
No it is not. It is actually 

"javageci-examples/src", "./src"

that is because the first is the directory when you run maven on the parent project and the second when you run maven in the child project as well as when you start the test from the IDE. This is the misunderstanding that explains the other change that is not ok.

Reply:
Since the explanation about maven() was:
1. Wrong
2. Ultimately pointless
It was removed.